cishouseholds/pipeline/high_level_transformations.py

```diff
@@ -1886,6 +1882,9 @@ def union_dependent_derivations(df):
         record_changed_column="cis_covid_vaccine_received",
         record_changed_value="Yes",
     )
+    df = df.withColumn(
```

Comment:
I think this needs moving to the generic processing, so that it gets applied before the lookup based editing? This should resolve the issues you're seeing in the regression test

Reply:
Fixed this and reran regression tests